src/lsm/tree_fuzz.zig

```diff
@@ -815,7 +873,7 @@ pub fn main(fuzz_args: fuzz.FuzzArgs) !void {
     };
 
     const fuzz_op_count = @min(
-        fuzz_args.events_max orelse @as(usize, 1E7),
+        fuzz_args.events_max orelse events_max,
```

Comment:
Maybe `events_max_default`?

Reply:
went back and forth here, but I think events_max is better.

This is both: 

* a compile-time limit on the number of events (so, `events_max`)
* a default runtime-limit for the actual number of events (so, `events_limit_default`)


comptime semantics is what I am relying on here